__tests__/app.test.js

```diff
@@ -167,6 +167,55 @@ describe('GET /api/articles/:article_id/comments', () => {
     })
 })
 
+describe('POST /api/articles/:article_id/comments', () => {
+    test('POST 201 adds a comment to the database when given a correctly formatted body and correct article id and returns the posted comment', () => {
+        return request(app)
+        .post('/api/articles/2/comments')
+        .expect(201)
+        .send({username: "butter_bridge", body: "test comment"})
+        .then(({body}) => {
+            expect(body.comment).toHaveProperty('author', "butter_bridge")
+            expect(body.comment).toHaveProperty('body', "test comment")
+        })        
```

Comment:
We have an article with no associated comments, which could use a test. In this case we would expect 201 responds with empty array of comments.